### ISSUE TITLE
Example of .env file

### ISSUE THREAD

**Issue by CodeLearner1234** [mikealmond/hq-trivia-assistant#53]
I am having trouble making my .env file can I have an example of what one looks like.
Thank You

**Comment by ps-one** [comment]
HQ_BEARER_TOKEN=
HQ_USER_ID=

**Comment by rizzryan** [comment]
To make the .env file, type cp .env.dist .env

**Comment by CodeLearner1234** [comment]
I am looking in finder and there is no .env file so I should use the command cp name.env.dist index.env ?

**Comment by rizzryan** [comment]
No. The . indicates that it is a hidden file. Download a text editor like Atom.io and open the project there.

**Comment by JChazzer** [comment]
Is the problem not that the 'cp' isn't recognised as a command on Windows terminal, and instead should be 'copy'?

**Comment by CodeLearner1234** [comment]
Also when I run the cp command it says no such directory or file

**Comment by JChazzer** [comment]
What operating system are you using? If it is Windows, instead of 'cp', do 'copy'.

**Comment by CodeLearner1234** [comment]
I am on macOS

**Comment by JChazzer** [comment]
Have you used the provided commands to install the components with Composer?

**Comment by CodeLearner1234** [comment]
I believe I have is their a way to confirm that I have done it correctly

**Comment by JChazzer** [comment]
When the install happened, some additional files to the original should have been installed.

You should have a file named .env.dist ?

**Comment by CodeLearner1234** [comment]
I don’t I am going to install from the start again

**Comment by CodeLearner1234** [comment]
Alright, it seemed to work this time. However, I am using Charles and I have found my Bearer but what is my ID?

**Comment by JChazzer** [comment]
Great. Your Bearer ID should be a 396-character long selection of letters and numbers, under the section 'Authentication'.

**Comment by CodeLearner1234** [comment]
Nevermind I believe I found it. It is just numbers correct?

**Comment by Samarth26** [comment]
I am having the same trouble on the mac. Is there a real solution to the problem?
